frontend/styles/globals.css

```
@@ -13,4 +13,49 @@ a {
 
 * {
   box-sizing: border-box;
+} */
+
+@font-face {
+  font-family: 'Montserrat-ExtraBold';
+  font-weight: 400;
+  src: url('../public/fonts/Montserrat/Montserrat-ExtraBold.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Montserrat-Bold';
+  font-weight: 700;
+  src: url('../public/fonts/Montserrat/Montserrat-Bold.ttf') format('truetype');
 }
+
+@font-face {
+  font-family: 'Montserrat-Light';
+  font-weight: 700;
+  src: url('../public/fonts/Montserrat/Montserrat-Bold.ttf') format('truetype');
+}
+
+
+@font-face {
+    font-family: 'LoraFonts';
+    font-weight: 900;
+    src: url('../public/fonts/11049.ttf') format('truetype');
+  }
+
+  .fonts-title_popular-block {
+    font-family:'Montserrat-ExtraBold';
+    opacity: 1;
+    color:#7B7C7B;
+    font-size: 39px;
+  }
+
+  .fonts-title-desc_popular_block {
+    font-family:'Montserrat-Bold';
```

По-хорошему, нужно после специфичного шрифта указывать общее название семейства шрифтов, которое будет применено в случае, если первый шрифт недоступен.
https://developer.mozilla.org/ru/docs/Web/CSS/font-family#%D1%81%D0%B8%D0%BD%D1%82%D0%B0%D0%BA%D1%81%D0%B8%D1%81
```suggestion
    font-family: 'Montserrat-Bold', serif;
```